src/polytope_oracles.jl

```diff
@@ -130,6 +130,30 @@ function convert_mathopt(
     return MathOptLMO(optimizer)
 end
 
+
+"""
+    ScaledBoundLInfNormBall(lower_bounds, upper_bounds)
+
+Polytope similar to a L-inf-ball with shifted bounds or general box constraints.
+Lower- and upper-bounds are passed on as abstract vectors, possibly of different types.
+For the standard L-inf-ball, all lower- and upper-bounds would be -1 and 1.
+"""
+struct ScaledBoundLInfNormBall{T, VT1 <: AbstractVector{T}, VT2 <: AbstractVector{T}} <: LinearMinimizationOracle
+    lower_bounds::VT1
+    upper_bounds::VT2
+end
+
+function compute_extreme_point(lmo::ScaledBoundLInfNormBall, direction; kwargs...)
+    v=lower_bounds
+    for i in eachindex(direction)
+        if direction[i]*upper_bounds[i] < direction[i]*lower_bounds[i]
+            v[i]=upper_bounds[i]
+        end
```

Comment:
I'm not sure about the product, does it work?
```suggestion
        if direction[i] * lmo.upper_bounds[i] < direction[i] * lmo.lower_bounds[i]
            v[i] = lmo.upper_bounds[i]
        end
```

Reply:
I think it should. The extreme points of the polytope are simply all combinations we can get from the entries of upper and lower bounds, so we can just iterate through all indices and always choose the according upper or lower bound depending on which one minimizes the scalar product, no?